pkg/pool-manager/pool_test.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package pool_manager
 
 import (
+	multus "github.com/intel/multus-cni/types"
```

Comment:
We usually split imports to 3 groups (sometimes 4 on test modules). The order is:

1. Built-in libraries ("net")
2. Third-party imports ("k8s.io/api/core/v1")
3. Internal imports ("github.com/k8snetworkplumbingwg/kubemacpool/pkg/names")

And then we sometimes plug a separate group with the test imports ("github.com/onsi/gomega") somewhere, but I can't really say we have a specific place for it.

By keeping this order, it is way easier for a reader of the code to quickly figure out what are the dependencies.

Reply:
ack